### PR TITLE
compose v2.18.0 release note

### PR DESCRIPTION
**Pull request by ndeloof**
### Proposed changes

release note for compose v2.18.0 release

### CODE REVIEW

File: compose/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,38 @@ redirect_from:
 ---
 {% include compose-eol.md %}
 
+## 2.18.0
+{% include release-date.html date="2023-05-16" %}
+
+### Enhancements
+- Dry run support
+- First (alpha) implementation of viz subcommand
```
Comment (aevesdocker) [inline compose/release-notes.md:16]:
```suggestion
- Added the first (alpha) implementation of viz subcommand
```

File: compose/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,38 @@ redirect_from:
 ---
 {% include compose-eol.md %}
 
+## 2.18.0
+{% include release-date.html date="2023-05-16" %}
+
+### Enhancements
```
Comment (aevesdocker) [inline compose/release-notes.md:14]: Let's combine `bug fixes` and `enhancements` to keep it consistent with previous release notes

File: compose/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,38 @@ redirect_from:
...
+- Dry run support
+- First (alpha) implementation of viz subcommand
+- introduce `--no-path-resolution` to skip relative path to be resolved
```
Comment (aevesdocker) [inline compose/release-notes.md:17]:
```suggestion
- Introduced `--no-path-resolution` to skip the relative path to be resolved
```

File: compose/release-notes.md [hunk 4]
```diff
@@ -8,6 +8,38 @@ redirect_from:
...
+- introduce `COMPOSE_ANSI` to define `--ansi` default value
```
Comment (aevesdocker) [inline compose/release-notes.md:18]:
```suggestion
- Introduced `COMPOSE_ANSI` to define the `--ansi` default value
```

File: compose/release-notes.md [hunk 5]
```diff
@@ -8,6 +8,38 @@ redirect_from:
...
+- introduce `COMPOSE_STATUS_STDOUT` to get status messages sent to stdout
```
Comment (aevesdocker) [inline compose/release-notes.md:19]:
```suggestion
- Introduced `COMPOSE_STATUS_STDOUT` which send the status messages to stdout
```

File: compose/release-notes.md [hunk 6]
```diff
@@ -8,6 +8,38 @@ redirect_from:
...
+
+### Bug fixes
```
Comment (aevesdocker) [inline compose/release-notes.md:21]: Are there specific PRs we can add as we do in previous release notes?

Comment (ndeloof) [reply compose/release-notes.md:21]: is this a requirement ? 
This is a synthesis for https://github.com/docker/compose/releases/tag/v2.18.0, which includes individual links

Comment (aevesdocker) [reply compose/release-notes.md:21]: It's just how it's always been done for Compose and how we do Desktop. But agree that it is a little silly to repeat all the links. Perhaps we can add something like 'For more detailed information, see [the release notes in the Compose repo](https://github.com/docker/compose/releases/tag/v2.18.0) and add that to line 13?

**Review by aevesdocker — APPROVED** [review]
Thank you @ndeloof

**Pull request merged**